ansible/group_vars/wakuv2-staging.yml

```diff
@@ -0,0 +1,105 @@
+  - { name: anthony, uid: 8004, groups: ['docker', 'dockremap'], key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDYM50p0dQDL2uZFtAHrpdf5sBLTrTLMZGABZosO2GTMu67GnvYeBn/L42hehhVX6yNDU4KOhZVWjLLI4Ekdex3dKf7ynLCSC1jVkR1k7YgVvVAwXCEk38MMe1LqUJmb2K0rRzlIn0LNThWawahAhGNLNU55nJR0X3MhvL7G3UnqGY1ubTABQGFkRPIlC0smuVFyRPmpLF2pGMNTyCkFpqAP7qu+wW3PguxDZRUpgjsM/iZtaHh1a8z7ikPGWhkvJkx/U0KtYuhoMbBPR/3Ta6ttDNXs8z/cOvc5cB6fBa9Y1IapfXCC8r88WsIkaN9F+7fTLdhN7+RJvHgUrxNzhLOPzvGd4KmHvE22goCrO9yZbxzxlRhZf6zgu27waU2CQhCDYXTASUcWzJgfSocxPCYPHYFeDFj9Vj8NqyAFSqEnXCFcz/nuzBAgKyEmwcbg+6uz6UfC7Fygz5f8xKerggP2tuwasEwHZkFsicc/NQZknWFkjvwT9sOqHrAmFlRYXbR8CzcsLHDMN/t5j333Akso0AuOVA2n8LvhEUmfrOI2Z9KM9Vd7uZdDXYtdYpRgUgFJCfw7KH3/cskJ8s9HH8AxbQ2KkjNB+Dv4ZWGthAkiLRbu/uxunHVjorkYZeJ76JuY5AxOd1OYvl2LTA/lxs4vEd2FVvw4AuBH+RImofd4w== anthony@status.im' }
+  - { name: daniel,  uid: 8005, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOPJlQ9iYVD2SNuW2ob2nvNHs+/O9PbcPc+dIepHNHO0 danielkaiser@status.im' }
+  - { name: ivan,    uid: 8006, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJBdm8y1PfWjT1pioaWJSZ2ETrUySb+dS/ifDg+VIpLY ivansete@status.im' }
+
+# Tag dependent on fleet: test, prod, staging
+nim_waku_cont_tag: 'deploy-wakuv2-{{ stage }}'
+nim_waku_cont_name: 'nim-waku-v2'
+nim_waku_log_level: 'debug'
+nim_waku_protocols_enabled: ['archive', 'relay', 'store', 'filter', 'lightpush', 'rln-relay', 'peer-exchange']
+nim_waku_disc_v5_enabled: true
+nim_waku_dns4_domain_name: '{{ dns_entry }}'
+nim_waku_node_key: '{{lookup("bitwarden", "fleets/wakuv2/"+stage+"/nodekeys", field=hostname)}}'
+
+# Topic configuration
+nim_waku_topics:
+  - '/waku/2/rs/18/16'
+  - '/waku/2/rs/18/17'
+  - '/waku/2/rs/18/18'
+  - '/waku/2/rs/18/128'
+  - '/waku/2/rs/18/129'
+  - '/waku/2/rs/18/130'
+  - '/waku/2/rs/18/768'
+  - '/waku/2/rs/18/769'
+  - '/waku/2/rs/18/770'
+  - '/waku/2/rs/18/896'
+  - '/waku/2/rs/18/897'
+  - '/waku/2/rs/18/896'
+
+
+# Ports
+nim_waku_p2p_tcp_port: 30303
+nim_waku_p2p_udp_port: 30303
+nim_waku_metrics_port: 8008
+nim_waku_disc_v5_port: 9000
+nim_waku_websock_port: 8000
+nim_waku_rpc_tcp_port: 8545
+
+# Limits
+nim_waku_p2p_max_connections: 300
+
+# Store
+nim_waku_store_message_retention_policy: 'time:2592000' # 30 days
+
+# DNS Discovery
+nim_waku_dns_disc_enabled: true
+nim_waku_dns_disc_url: 'enrtree://AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM@test.waku.nodes.status.im'
```

Comment:
Is this intentionally referencing test fleet? Or simply because we have no DNS entries for staging fleet yet?

Reply:
Hanno says we will want DNS discovery entries for this fleet, so probably the correct solution for this fleet is to disable DNS discovery for now and add it later.

Reply:
I'll remove it for now.

Reply:
Yes, this would create a bit of a hurdle as we'll need the hosts to run before we can scrape their signed ENRs and construct the tree for DNS discovery. My suggestion: launch the fleet with DNS disc disabled, construct and publish the tree and then update config.